### PR TITLE
[fix] loss func return torch.tensor for new megatron compatibility

### PR DESCRIPTION
**Pull request by yueming-yuan**
return num_tokens as torch.tensor(1) rather than int(1)
Newer Megatron require receiving num_tokens as a torch tensor for later processing
location: `miles/backends/megatron_utils/loss.py`

### CODE REVIEW

File: miles/backends/megatron_utils/loss.py [hunk 1]
```diff
@@ -771,7 +771,7 @@ def loss_function(
 
     return (
         loss,
-        num_tokens if args.calculate_per_token_loss else 1,
+        num_tokens if args.calculate_per_token_loss else torch.tensor(1, dtype=torch.int, device=logits.device),
```
Comment (fzyzcjy) [inline miles/backends/megatron_utils/loss.py:774]: qq: does the `if` branch give tensor or not (seems your title says should give)
